bot/worker.py

```diff
@@ -109,76 +110,73 @@ def process_body(body_text):
+    """Method that tries to add a author's flair specific reply to the comment/submission.
 
-    return body_text
-
-
-def do_flair_specific_reply(thing, response):
-    hero_id = db_api.get_hero_id_by_flair_css(flair_css=thing.author_flair_css_class)
+    :param replyable: The comment/submission on reddit
+    :param processed_text: The processed body text
+    :return: True if the replyable was replied to, else False.
+    """
+    hero_id = db_api.get_hero_id_by_flair_css(flair_css=replyable.author_flair_css_class)
     if hero_id:
-        link, hero_id = db_api.get_link_for_response(
-            response=response, hero_id=hero_id)
+        link, hero_id = db_api.get_link_for_response(processed_text=processed_text, hero_id=hero_id)
         if link:
-            reply = create_reply(thing=thing, response_url=link, hero_id=hero_id)
-            thing.reply(reply)
-            logger.info("Added: " + thing.fullname)
+            reply = create_reply(replyable=replyable, response_url=link, hero_id=hero_id)
+            replyable.reply(reply)
+            logger.info("Added: " + replyable.fullname)
             return True
+    return False
 
 
-def do_regular_reply(thing, response):
-    """Method to create response for given thing.
+def do_regular_reply(replyable, processed_text):
+    """Method to create response for given replyable.
     In case of multiple matches, it used to sort responses in descending order of heroes, but now it's random.
-    add_shitty_wizard_response was very similar to this, and hence has been merged
 
-    :param thing: The thing/submission on reddit
-    :param response: The plaintext processed thing body
+    :param replyable: The comment/submission on reddit
+    :param processed_text: The processed body text
     :return: None
     """
 
-    link, hero_id = db_api.get_link_for_response(processed_text=response)
+    link, hero_id = db_api.get_link_for_response(processed_text=processed_text)
 
     if link and hero_id:
         img_dir = db_api.get_img_dir_by_id(hero_id=hero_id)
 
-        thing.reply(create_reply(thing=thing, response_url=link, hero_id=hero_id, img=img_dir))
+        replyable.reply(create_reply(replyable=replyable, response_url=link, hero_id=hero_id, img=img_dir))
 
-        logger.info("Replied to: " + thing.id)
+        logger.info("Replied to: " + replyable.id)
 
 
-def create_reply(thing, response_url, hero_id, img=None):
+def create_reply(replyable, response_url, hero_id, img=None):
     """Method that creates a reply in reddit format.
 
-    The message consists of a link the the response, the response itself, a warning about the sound
-    and an ending added from the config file (post footer). Image is currently ignored due to new reddit not
-    rendering flairs properly.
-    """
-    original_text = thing.body if isinstance(thing, Comment) else thing.title
+    The message consists to a link the the response audio file, the response itself, a warning about the sound
```

Comment:
'link the the response' -> 'link to the response'

Reply:
Fixed in 9895807a3ac442eb4ac325e26b4937e203177fe3